src/prelogin-payload.ts

```diff
@@ -180,6 +187,17 @@ class PreloginPayload {
     };
   }
 
+  createTraceIdOption() {
+    const buffer = new WritableTrackingBuffer(traceIdSize);
+    // generate a random series of bytes to use as the TraceID
+    // used for debugging purposes
+    buffer.writeBuffer(randomBytes(traceIdSize));
```

Comment:
for now the traceID is just random. We could consider actually making it based on PID or at least persistent through the session, but I think it's fine if it's always just a unique ID.